src/absil/ilreflect.fs

```diff
@@ -324,25 +324,7 @@ type cenv =
 // [ns]            ,name -> ns+name
 // [ns;typeA;typeB],name -> ns+typeA+typeB+name
 let convTypeRefAux (cenv:cenv) (tref:ILTypeRef) = 
-
-    // If an inner nested type's name contains a space, the proper encoding is "\+" on both sides - otherwise,
-    // we use "+"
```

Comment:
That code had always been very, very suspect to me.  Thanks for digging into this so thoroughly